lib/docs/src/components/table.stories.tsx

```diff
@@ -267,6 +267,18 @@ export const SelectableMultiple = (): JSX.Element => {
 	);
 };
 
+Utils.story(SelectableMultiple, {
+	desc: `
+To choose multiple rows from the table, user can use the \`selectable\` prop.
+If it is defined, a [checkbox][1] will be displayed on each row. This prop requires an object 
+that contain the key of selected rows and a callback to set selected rows. 
+These requirement can be achieved via [the State Hook][2].
```

Comment:
Please review the implementation :D the checkbox (multi-select) or radio (single-select) is based on the type of "selected" and "setSelected".